### PR TITLE
Added simple demo config

### LINKED ISSUE #9: Create example for pg_datanymizer
Create example to introduce how to get starting with CLI tool `pg_datanymizer`.
Setup environment with docker-compose, test database dump and example config file.

### PR DESCRIPTION
**Pull request by akirill0v**
Added simple demo config and docker-compose example for starting experimentation.

Close #9. 

#### ✓ Checklist:

- [x] This PR has been added to [CHANGELOG.md](https://github.com/datanymizer/datanymizer/blob/master/CHANGELOG.md) (at the top of the list);
- [ ] Tests for the changes have been added (for bug fixes / features);
- [x] Docs have been added / updated (for bug fixes / features).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,8 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### 🚀 Added
 - Add the uniqueness option for email, ip and random_num [#16](https://github.com/datanymizer/datanymizer/pull/16)
   ([@evgeniy-r](https://github.com/evgeniy-r))
+- Added demo configuration end docker-compose example [#20](https://github.com/datanymizer/datanymizer/pull/20)
```
Comment (mgrachev) [inline CHANGELOG.md:11]:
```suggestion
- Add demo configuration and docker-compose example [#20](https://github.com/datanymizer/datanymizer/pull/20) ([@akirill0v](https://github.com/akirill0v))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,8 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### 🚀 Added
 - Add the uniqueness option for email, ip and random_num [#16](https://github.com/datanymizer/datanymizer/pull/16)
   ([@evgeniy-r](https://github.com/evgeniy-r))
+- Added demo configuration end docker-compose example [#20](https://github.com/datanymizer/datanymizer/pull/20) 
 
 ### ⚙️ Changed
+([@akirill0v](https://github.com/akirill0v))
```
Comment (mgrachev) [inline CHANGELOG.md:14]:
```suggestion
```

**Comment by mgrachev** [comment]
I think we should rename the `demo/README.md` file to `CONTRIBUTING.md`.

**Comment by akirill0v** [comment]
@mgrachev Thanks for your changes! I thought about your proposal and will make a text for contributing in future commits.

**Review by mgrachev — APPROVED** [review]

**Pull request merged**